scripts/metavar.py

```diff
@@ -1184,7 +1185,7 @@ def equiv(self, vmatch):
         of the objects.
         equiv should be overridden with a method that first calls this
         method and then tests class-specific object data."""
-        return vmatch.__class__ == self.__class__
+        return vmatch.__class__ != self.__class__
```

Comment:
This was a (bad) hack for _add_to_list to work. 
Now that I'm thinking about this considering "multiple actions for each variable", there will be some more changes coming. This will go away